### DISCUSSION TITLE
[@types/chrome] - Property 'open' does not exist on type 'typeof sidePanel'.

### DISCUSSION THREAD

**Discussion by StratusBase** [DefinitelyTyped/DefinitelyTyped#66493] [Issues with a @types package]
[@types/chrome] Version 0.0.244

While trying to use the following `chrome.sidePanel.open()` - per:
https://developer.chrome.com/docs/extensions/reference/sidePanel/#method-open

I get the following error in TypeScript:
Property 'open' does not exist on type 'typeof sidePanel'.

[a comment by an automated account is not shown]

**Comment by HowieG** [comment]
Have this same issue

**Comment by erwanjugand** [comment]
Relates to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67146
